src/goto-symex/path_storage.h

```diff
@@ -22,8 +22,12 @@
 class symex_nondet_generatort
 {
 public:
+  DEPRECATED(SINCE(
+    28, 05, 2019, "Use operator()(typet, source_locationt) instead"))
   nondet_symbol_exprt operator()(const typet &type);
```

Comment:
I think it's ok to just remove this without a deprecation path. It's not even that long ago that this code has been introduced in the first place, and no one bothered to actually do this in a way that would maintained API compatibility.